uspv/init.go

```diff
@@ -73,32 +74,32 @@ func (s *SPVCon) GetListOfNodes() ([]string, error) {
 }
 
 // DialNode receives a list of node ips and then tries to connect to them one by one.
-func (s *SPVCon) DialNode(listOfNodes []string) error {
```

Comment:
We _might_ use DialNode in other places in the future (see: Tor routing PR), so lets leave this exported :)